astropy/utils/data_info.py

```diff
@@ -31,8 +31,8 @@
 from ..extern.six.moves import zip, cStringIO as StringIO
 
 # Tuple of filterwarnings kwargs to ignore when calling info
-IGNORE_WARNINGS = (dict(category=RuntimeWarning,
-                        module=r'numpy\.lib\.nanfunctions'),)
+IGNORE_WARNINGS = (dict(category=RuntimeWarning, message='All-NaN|'
+                        'Mean of empty slice|Degrees of freedom <= 0'),)
```

Comment:
I generally prefer the `dict` literal:

```
IGNORE_WARNINGS = {'category': RuntimeWarning, 
                   'message': 'All-NaN|Mean of empty slice|Degrees of freedom <= 0'}
```

But that's probably very subjective. :sweat_smile: